packages/store/src/utils/index.ts

```diff
@@ -1,5 +1,6 @@
 import { homedir } from 'node:os';
 import path from 'node:path';
```

Comment:
Should you be using the named import here? i.e. `import { join } from 'node:path';`